pkgs/development/libraries/newt/default.nix

```diff
@@ -27,6 +36,13 @@ stdenv.mkDerivation rec {
     "CROSS_COMPILE=${stdenv.cc.targetPrefix}"
   ];
 
+  outputs = [ "out" ] ++ lib.optional pythonSupport "python";
```

Comment:
hmm is this the normal way to do optional python libraries? I've never seen it like this before. If this is an established pattern it's fine, but if it's your invention maybe make it something like `pythonLibrary`? an attribute called `python` seems to me like it should contain a python interpreter

Reply:
A quick `rg -F '.python;'` gives a few results in `pkgs/top-level/python-packages.nix`.
I copied this from an older PR #22774 opened for Byobu and changed it from `.py` to `.python`.

There is no instance of `.pythonLibrary`in the entirety of nixpkgs.

Reply:
I would be for not having a separate input flag, and just always create a `python` output.

There's no conditional configure steps, and the python assets are always created as part of the build anyway.

I don't think it's worth having to do `newt.override { pythonSupport = true; }` to get the output which was "free" to begin with.

Reply:
I almost asked for that, but didn't feel confident in doing so.